### PR TITLE
[cli] Update `new` cli for latest flipper

### PR DESCRIPTION
**Pull request by shawntabrizi**
- Fixes the test in the example contract
- Changes some hard coded instances of "Flipper" to use the desired variable name
- Calls println when the getter is called

Tested on MacOS

### CODE REVIEW

File: cli/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "cargo-contract"
 version = "0.1.0"
-authors = ["Robin Freyler <robin@parity.io>", "Parity Technologies <admin@parity.io>"]
+authors = ["Parity Technologies <admin@parity.io>"]
```
Comment (Robbepop) [inline cli/Cargo.toml]: I don't understand this change since I thought that this is already in master, isn't it?

File: cli/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Cargo plugin for Ink contracts
+
+A small CLI tool for helping setting up and managing WebAssembly smart contracts written with ink!.
```
Comment (Robbepop) [inline cli/README.md]: Same here ...

File: cli/src/cmd/new.rs [hunk 3]
```diff
@@ -128,25 +132,26 @@ contract! {{
...
+            println(&format!("Storage Value: {{:?}}", *self.value));
             *self.value
         }}
     }}
 }}
 
 #[cfg(test)]
 mod tests {{
-    use super::Flipper;
+    use super::{};
```
Comment (Robbepop) [inline cli/src/cmd/new.rs]: maybe you mean to be using `use super::*;` here?

File: cli/src/cmd/new.rs [hunk 4]
```diff
@@ -128,25 +132,26 @@ contract! {{
...
-        assert_eq!(flipper.get(), false);
+        let mut contract = {}::deploy_mock();
+        assert_eq!(contract.get(), false);
+        contract.flip();
+        assert_eq!(contract.get(), true);
     }}
 }}
 "##,
-        camel_name, camel_name, camel_name,
+        camel_name, camel_name, camel_name, camel_name, camel_name,
```
Comment (Robbepop) [inline cli/src/cmd/new.rs]: lol

File: cli/src/cmd/new.rs (hunk 3 above)
Comment (shawntabrizi) [reply cli/src/cmd/new.rs]: That is probably more generic right? Can change.

File: cli/Cargo.toml (hunk 1 above)
Comment (shawntabrizi) [reply cli/Cargo.toml]: ¯\_(ツ)_/¯

File: cli/src/cmd/new.rs (hunk 3 above)
Comment (shawntabrizi) [reply cli/src/cmd/new.rs]: Resolved

File: cli/Cargo.toml (hunk 1 above)
Comment (Robbepop) [reply cli/Cargo.toml]: Yeah so at the "Converation" tab it says that this branch is out of date and you should update it. Please do so. :S

File: cli/src/cmd/new.rs [hunk 5]
```diff
@@ -161,12 +166,15 @@ fn build_sh_contents(name: &str) -> String {
...
-CARGO_INCREMENTAL=0 &&
-cargo build --release --features generate-api-description --target=wasm32-unknown-unknown --verbose &&
-wasm2wat -o target/$PROJNAME.wat target/wasm32-unknown-unknown/release/$PROJNAME.wasm &&
-cat target/$PROJNAME.wat | sed "s/(import \"env\" \"memory\" (memory (;0;) 2))/(import \"env\" \"memory\" (memory (;0;) 2 16))/" > target/$PROJNAME-fixed.wat &&
-wat2wasm -o target/$PROJNAME.wasm target/$PROJNAME-fixed.wat &&
-wasm-opt -Oz target/$PROJNAME.wasm -o target/$PROJNAME-opt.wasm"##,
+# cargo clean
+# rm Cargo.lock
+
+CARGO_INCREMENTAL=0 cargo build --release --features generate-api-description --target=wasm32-unknown-unknown --verbose
```
Comment (Robbepop) [inline cli/src/cmd/new.rs]: Why have you removed `&&` here?
I think our approach was to add `set -e`, wasn't it?

Comment (shawntabrizi) [reply cli/src/cmd/new.rs]: I copied what was in the build.sh script from the examples. Can you `suggest` an improvement to this?

**Comment by shawntabrizi** [comment]
@ascjones Will try to help get this fixed...

File: cli/src/cmd/new.rs (hunk 5 above)
Comment (Robbepop) [reply cli/src/cmd/new.rs]: Just add `set -e` on top of the script for now

**Pull request closed without merging**